__tests__/app.test.js

```diff
@@ -0,0 +1,39 @@
+const request = require("supertest");
+const app = require("../app");
+const connection = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data");
+const endpoints= require('../endpoints.json')
+
+beforeEach(() => {
+  return seed(testData);
+});
+
+afterAll(() => connection.end());
+
+describe("/api", () => {
+  test("GET - status:200 - returns JSON with available endpoints ", () => {
+    return request(app)
+      .get("/api")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.endpoints).toEqual(endpoints);
+      });
+    });
+});
+
+describe("/api/categories", () => {
```

Comment:
Looks like some of task 3 has ended up on this PR. Be mindful not to mix up your tasks on the same branch